Makefile

```diff
@@ -40,34 +46,58 @@ $(OBJECT_DIR)/libArachne.a: $(OBJECTS)
 
 -include $(DEP)
 
+$(OBJECT_DIR)/CArachneWrapperCTest.d: $(WRAPPER_DIR)/CArachneWrapperCTest.c | $(OBJECT_DIR)
```

Comment:
It looks like you can drop the prefixes on this line through line 60, and the Makefile will still work correctly.

Reply:
Yes it works. 👍